create_pnf_service_template.py

```diff
@@ -11,15 +11,27 @@ def pair(s):
         raise argparse.ArgumentTypeError("Physical pairs must be in the form a1,b1 a2,b2 a3,b3 ...")
 
 parser = argparse.ArgumentParser(description='Create service template and link PIs')
-parser.add_argument('--st_name', help='service template name', required=True)
-parser.add_argument('--physical_pairs', nargs='*', type=pair, help='Physical pairs must be in the form a1,b1 a2,b2 a3,b3 ...)', required=True)
+parser.add_argument('--service_template_name', help='service template name', required=True)
+parser.add_argument('--physical_interface_pairs', nargs='*', type=pair, help='Physical pairs must be in the form a1,b1 a2,b2 a3,b3 ...)', required=True)
+parser.add_argument('--admin_user', help='optional admin username')
```

Comment:
You can add a default for a argument
parser.add_argument('--admin_user', help='optional admin username', default="admin")